apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts

```diff
@@ -728,4 +734,17 @@ export class AccountOperationsComponent implements OnInit, AfterViewInit, OnDest
             }
         });
     }
+
+    /**
+     * Export master file
```

Comment:
JS doc is not proper. Use DocumentThis extension in VScode and do proper documentation with correct message

Reply:
done